tools/nnyeah/nnyeah/nnyeah.csproj

```diff
@@ -26,4 +34,25 @@
       <DependentUpon>Errors.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+

```

Comment:
We should add the following: 
```csharp
<Error Text="Foo" Condition="!Exists('$(FilePath)')" /> 
```

to catch when they are not present (I don't know if _GetPackageFiles fails).

Reply:
It already fails if you screw up a path or it doesn't exist:

```
/usr/local/share/dotnet/sdk/6.0.201/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(221,5): error : Could not find a part of the path '/Users/donblas/Programming/xamarin-macios/tools/nnyeah/nnyeah/Microsoft.iOS.Ref/ref/net6.0'. [/Users/donblas/Programming/xamarin-macios/tools/nnyeah/nnyeah/nnyeah.csproj]
```